lib/templates/reconciliationText.js

```diff
@@ -153,6 +156,12 @@ class ReconciliationText {
     }, {});
   }
 
+  static #updateConfigIfChanged(originalConfig, currentConfig, handle) {
```

Comment:
[SUGGESTION] Should this item be moved to `fsUtils` as may also wish to use for other template types?

Reply:
Good remark. I though about it, and currently no other template writes the config when calling `read()`, only ReconciliationText. 

I would say that makes more sense actually, we shouldn't write a new config when just reading. In ReconciliationText we are performing some adjustments. We even consider getting rid of those write operations and stick to reading only. But from now on we will only write if changed (once prrobably?)